examples/DQMC/from_fcidump/n2/n2.py

```diff
@@ -0,0 +1,63 @@
+# N2 ground state
+
```

Comment:
I think this example is very similar to h2o in terms of workflow, if you want to add another example maybe add a uhf or non-zero spin one